app/src/main/java/org/raleighmasjid/iar/MainApplication.kt

```diff
@@ -1,7 +1,21 @@
 package org.raleighmasjid.iar
 
 import android.app.Application
+import com.onesignal.OneSignal
 import dagger.hilt.android.HiltAndroidApp
 
+const val ONESIGNAL_APP_ID = "01fcf852-7b3f-4b53-a733-4cb8241bd193"
+
```

Comment:
Should this be some kind of `env` variable? How can we handle this?

Reply:
just be careful, this repo is public so we don't want to commit sensitive keys to the repo. I don't think this app id is that sensitive. In terms of organization I would suggest maybe using a BuildConfig https://guides.codepath.com/android/Storing-Secret-Keys-in-Android#hidden-in-buildconfigs